src/pages/rien.js

```diff
@@ -0,0 +1,52 @@
+import { injectIntl } from 'gatsby-plugin-intl';
+import React from 'react';
+import PageTemplate from '../templates/pageTemplate';
+import Markdown from 'react-remarkable'
+import RienImage from '../components/IndexPage/rien.jpg';
+
+const Rien = ({ intl }) => {
+  const title = intl.formatMessage({ id: 'rien.title' });
+
+  return (
+    <PageTemplate title={title} useCustomStructure={true}>
+      <div className='bg-dark text-white' style={{ minHeight: '100vh' }}>
+        <div
+          className='container d-flex flex-column align-center h-100'
+          style={{
+            minHeight: '100vh',
+            paddingTop: '50px',
+            paddingBottom: '50px',
+          }}
+        >
+          <div className='my-auto text-center'>
+            <h2 className='mt-5 mb-3'>
+              {intl.formatMessage({ id: 'rien.title' })}
+            </h2>
+            <p className='lead'></p>
+            <h4 className='mb-3'>Rien Fröhlich</h4>
```

Comment:
You can remove this line, it says "Rien Fröhlich" twice on the page now, because `rien.title` also contains "Rien Fröhlich"